packages/amplify_authenticator/lib/src/utils/country_code.dart

```diff
@@ -270,3 +271,13 @@ const countryCodes = [
   Country('ZM', '260', CountryResolverKey.zm$),
   Country('ZW', '263', CountryResolverKey.zw$)
 ];
+
+Country? findDeviceLocationFromList() {
+  try {
+    return countryCodes.firstWhere((country) =>
+        country.countryCode ==
+        WidgetsBinding.instance?.window.locale.countryCode);
```

Comment:
Should we use `Localizations.localeOf(context).countryCode`? I think `WidgetsBinding.instance?.window.locale.countryCode` always grabs from their device, even if they have specified a different value for the app. I would assume  `Localizations.localeOf(context).countryCode` falls back to the device setting if a value wasn't explicitly set, although I am not positive.